ui/components/src/ContentWithLegend/model/content-with-legend-model.ts

```diff
@@ -112,16 +127,26 @@ export function getContentWithLegendLayout({
     return noLegendLayout;
   }
 
-  const { position } = legendOptions;
-
-  const legendWidth = position === 'Right' ? 200 : width;
-
-  // TODO: account for number of legend items returned when adjusting legend spacing
-  let legendHeight = LEGEND_HEIGHT_SM;
-  if (position === 'Right') {
-    legendHeight = height;
-  } else if (height >= PANEL_HEIGHT_LG_BREAKPOINT) {
-    legendHeight = LEGEND_HEIGHT_LG;
+  const { position, mode } = legendOptions;
+
+  let legendWidth;
+  let legendHeight;
+
+  if (mode === 'List') {
+    // TODO: normalize list to share similar height options as the table
+    // when we add more size options.
```

Comment:
This avoids this being really disruptive to current list users who were used to the previous size. We default the table to what will be "medium" when we add the new sizes.